content/millennium/r4/diagnostic/observation.md

```diff
@@ -46,6 +46,8 @@ The following fields are returned if valued:
 * valueAttachment: URL for this extension is defined as: `http://hl7.org/fhir/5.0/StructureDefinition/extension-Observation.valueAttachment`
   This extension is defined and referenced from the newer version of FHIR. See [Extensions for converting between versions] and [R5 Snapshot of Observation.value] for more information.
 
+* [Performer Function]   
```

Comment:
i think we should name this performerFunction

Reply:
can we omit the blank line between bullets?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/387/commits/91dd92d1f4b98babbedbd7a20920106fd79ce752
![Screen Shot 2020-05-21 at 11 56 53 AM](https://user-images.githubusercontent.com/42872353/82584787-9c4ee700-9b5a-11ea-852e-2fed786d0af9.png)